src/mythos/AllEncounterCardsBag.ttslua

```diff
@@ -1,7 +1,7 @@
 require("core/Constants")
 require("playercards/PlayerCardSpawner")
 
-local cardIdIndex  = {}
+local objectIdIndex  = {}
```

Comment:
I think I would rather have the objectIdIndex as second index and keep the cards separate